src/types.ts

```diff
@@ -9,7 +9,8 @@ export type Env = {
     | D1Database
     | Fetcher
     | Hyperdrive
-    | DurableObjectNamespace;
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    | DurableObjectNamespace<any>;
```

Comment:
what about unknown instead of any ?

Reply:
The generic is constrainted so unknown dosent satisfy but Any gets a pass.